collector-http/src/main/java/zipkin2/collector/otel/http/SpanTranslator.java

```diff
@@ -50,29 +51,36 @@ final class SpanTranslator {
     this(DefaultOtelResourceMapper.create());
   }
 
-  List<zipkin2.Span> translate(ExportTraceServiceRequest otelSpans) {
+  List<zipkin2.Span> translate(ExportTraceServiceRequest otelSpans, boolean base64Decoded) {
     List<zipkin2.Span> spans = new ArrayList<>();
     List<ResourceSpans> spansList = otelSpans.getResourceSpansList();
     for (ResourceSpans resourceSpans : spansList) {
       Resource resource = resourceSpans.getResource();
       for (ScopeSpans scopeSpans : resourceSpans.getScopeSpansList()) {
         InstrumentationScope scope = scopeSpans.getScope();
         for (io.opentelemetry.proto.trace.v1.Span span : scopeSpans.getSpansList()) {
-          spans.add(generateSpan(span, scope, resource));
+          spans.add(generateSpan(span, scope, resource, base64Decoded));
         }
       }
     }
     return spans;
   }
 
+  // for backward compatibility
```

Comment:
with what?

Reply:
for existing test cases that used the old method.
Fixed in https://github.com/openzipkin-contrib/zipkin-otel/pull/31/commits/4b1a59a1dbb1a71de31719d7acbef83e4f77f368